Python/Product/PythonTools/PythonTools/Options/OptionsTreeView.cs

```diff
@@ -42,28 +39,16 @@ class OptionsTreeView : TreeView {
 
         private void InitializeImageList() {
             if (_imageList == null) {
-                using (var imgHandler = new ImageHandler(typeof(OptionsTreeView).Assembly.GetManifestResourceStream("Microsoft.PythonTools.Project.Resources.imagelis.bmp"))) {
-#pragma warning disable 0618
-                    var openFolder = imgHandler.ImageList.Images[(int)ProjectNode.ImageName.OpenFolder];
-                    var closedFolder = imgHandler.ImageList.Images[(int)ProjectNode.ImageName.Folder];
-#pragma warning restore 0618
-
-                    _imageList = new ImageList();
-
-                    Bitmap bmp = new Bitmap(16, 16);
-
-                    // transparent image is the image we use for owner drawn icons
-                    _imageList.TransparentColor = Color.Magenta;
-                    using (var g = Graphics.FromImage(bmp)) {
-                        g.FillRectangle(
-                            Brushes.Magenta,
-                            new Rectangle(0, 0, 16, 16)
-                        );
-                    }
-                    _imageList.Images.Add(bmp);
-                    _imageList.Images.Add(openFolder);
-                    _imageList.Images.Add(closedFolder);
+                _imageList = new ImageList();

```

Comment:
Oh I missed the static there, yeah, better not change it.  I agree about WPF treeview.  I know the high dpi support on these formatting options pages has been problematic at 200%+ and WPF.  I haven't ported the issues from codeplex as I don't have access to a machine that can reasonably go that high anymore.